.github/workflows/pr-build.yaml

```diff
@@ -206,11 +206,12 @@ jobs:
               sap-netweaver
               servicenow
               slack
-          - category: SQL
+          - category: Database
```

Comment:
If you rename these, you also need to tweak the references in the `matrix.category` array.

Reply:
Ok, I found it. Thanks for the tips @jamesnetherton